consensus/istanbul/validator/weighted.go

```diff
@@ -823,13 +823,14 @@ func calcTotalAmount(weightedValidators []*weightedValidator, stakingInfo *rewar
 }
 
 // calcWeight updates each validator's weight based on the ratio of its staking amount vs. the total staking amount.
-func calcWeight(weightedValidators []*weightedValidator, stakingAmounts []float64, totalStaking float64) {
+func calcWeight(weightedValidators []*weightedValidator, stakingAmounts []float64, totalStaking float64, rules params.Rules) {
 	localLogger := logger.NewWith()
 	if totalStaking > 0 {
 		for i, weightedVal := range weightedValidators {
 			weight := uint64(math.Round(stakingAmounts[i] * 100 / totalStaking))
```

Comment:
What if disabling this computation? We don't need this kind of multiply and division if Kore is enabled.

Reply:
Good Idea!
I did some tests and It was working.
Thank you for your review.
Reflected in `fa4f659e0e434a2601cae4008eff77060e254c91`.